pkg/controller/accountclaim/accountclaim_controller.go

```diff
@@ -140,7 +138,7 @@ func (r *ReconcileAccountClaim) Reconcile(request reconcile.Request) (reconcile.
 					// First we want to see if this was an update race condition where the credentials rotator will update the CR while the finalizer is trying to run.  If that's the case, we want to requeue and retry, before outright failing the account.
 					if errors.IsConflict(err) {
 						reqLogger.Info("Account CR Modified during CR reset.")
-						err = rawError.New(fmt.Sprintf("Account CR Modified during CR reset. %s", err.Error()))
+						err = fmt.Errorf("Account CR Modified during CR reset. %s", err.Error())
```

Comment:
`errors.Wrap` does pretty much the same thing, but includes more useful information.

Reply:
TIL, thanks!